src/FileUtility.ts

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+import * as fs from 'fs-plus';
+import * as path from 'path';
+import * as sdk from 'vscode-iot-device-cube-sdk';
+
+export class FileUtility {
+
+  // File-related functions to help create files on workspace path
+  static async existsInWorkspace(localPath: string): Promise<boolean> {
```

Comment:
How is this path related to workspace?

Reply:
I think the api signature should be changed to
fileExists(path, location).    If location is local, use the API from fs-plus, else use the API from cube.

Reply:
* **Workspace path**  looks like: `/workspace/myproject/devkit/sample.ino`
* **Local path** looks like: `C:/users/username/Documents/IoTWorkbenchProjects/projects/myproject/devkit/sample.ino`

I use `async` to align two sets of API(for local path and workspace path) which will save much duplicate code. And the local file-related functions is said will be refactored to be `async`. Did I misunderstand that?

Also, your suggestion to pass the path as an argument may be a better implementation. I will check whether this method can work well in the specific scenario.